adding run history infoto deploy jobs page (#3415)
dbt-labs/docs.getdbt.com: website/docs/docs/deploy/dbt-cloud-job.md

Refer to [build and view your docs](/docs/collaborate/build-and-view-your-docs) for more info.

### Run history

> The Run History dashboard in dbt Cloud helps you monitor the health of your dbt project. It provides a detailed overview of all of your project's job runs and empowers you with a variety of filters to help you focus on specific aspects. You can also use it to review recent runs, verify overnight results, or track the progress of running jobs. You can access it on the top navigation menu by clicking **Deploy** and then **Run History**. 

Reviewer: ```suggestion
The Run History dashboard in dbt Cloud helps you monitor the health of your dbt project. It provides a detailed overview of all of your project's job runs and empowers you with a variety of filters to help you focus on specific aspects. You can also use it to review recent runs, find errored runs, and track the progress of runs in progress. You can access it on the top navigation menu by clicking **Deploy** and then **Run History**. 
```

> The dashboard displays your full run history, including job name, status, environment associated, job trigger, commit SHA, schema and timing info. 

Author: ```suggestion
The dashboard displays your full run history, including job name, status, environment associated, job trigger, commit SHA, schema, and timing info. 
```

> From June 1, 2023, dbt Cloud multi-tenant users can access their run history for the last 365 days through the dbt Cloud user interface (UI) and API. This optimizes dbt Cloud's performance by reducing the amount of stored metadata. For more info, refer to [Older run history retrieval change](/docs/dbt-versions/release-notes/May-2023/run-history-endpoint)

Author: ```suggestion
From June 1, 2023, dbt Cloud multi-tenant users can access their run history for the last 365 days through the dbt Cloud user interface (UI) and API. This optimizes dbt Cloud's performance by reducing the amount of stored metadata. For more info, refer to `Older run history retrieval change /docs/dbt-versions/release-notes/May-2023/run-history-endpoint` will add link once pr 3407 merges later
```

> From June 1, 2023, dbt Cloud multi-tenant users can access their run history for the last 365 days through the dbt Cloud user interface (UI) and API. This optimizes dbt Cloud's performance by reducing the amount of stored metadata. For more info, refer to [Older run history retrieval change](/docs/dbt-versions/release-notes/May-2023/run-history-endpoint).

Reviewer: ```suggestion
dbt Cloud developers can access their run history for the last 365 days through the dbt Cloud user interface (UI) and API.

We limit self-service retrieval of run history metadata to 365 days in order improve dbt Cloud's performance. For more info on the run history retrieval change, refer to [Older run history retrieval change](/docs/dbt-versions/release-notes/May-2023/run-history-endpoint).
```

Author: thanks so much @schottj, great suggestions! i purposely left out the 'contact support to retrieve older run history data' because:

a. there is a deadline for when support can do this (until sept). this means we may have outdated content and will need to manually update this doc to remove this portion.
b. this specific use case may not apply to all users (i think you said only a couple of dozen) so i don't think it's beneficial for users to see this or know this. they can always read more by going to the release note.
c. the couple of dozen users that want their old run history data will get an email and have until June 1st to request this. 

what do you think of the below:

> dbt Cloud developers can access their run history for the last 365 days through the dbt Cloud user interface (UI) and API. We limit self-service retrieval of run history metadata to 365 days in order to improve dbt Cloud's performance. For more info on the run history retrieval change and requesting older history data, refer to [Older run history retrieval change](/docs/dbt-versions/release-notes/May-2023/run-history-endpoint)

Reviewer: that looks good to me

> From June 1, 2023, dbt Cloud multi-tenant users can access their run history for the last 365 days through the dbt Cloud user interface (UI) and API. This optimizes dbt Cloud's performance by reducing the amount of stored metadata. For more info, refer to `Older run history retrieval change /docs/dbt-versions/release-notes/May-2023/run-history-endpoint` will add link once pr 3407 merges later

Author: ```suggestion
From June 1, 2023, dbt Cloud multi-tenant users can access their run history for the last 365 days through the dbt Cloud user interface (UI) and API. This optimizes dbt Cloud's performance by reducing the amount of stored metadata. For more info, refer to [Older run history retrieval change](/docs/dbt-versions/release-notes/May-2023/run-history-endpoint).
```

> <Lightbox src="/img/docs/dbt-cloud/deployment/run-history.jpg" width="85%" title="Run History dashboard allow you to monitor the health of your dbt project and displays jobs, job status, environment, timing, and more."/>

Reviewer: ```suggestion
<Lightbox src="/img/docs/dbt-cloud/deployment/run-history.jpg" width="85%" title="Run History dashboard allows you to monitor the health of your dbt project and displays jobs, job status, environment, timing, and more."/>
```
